test/tempfile.go

```diff
@@ -20,23 +20,24 @@ func (s *TempFileSuite) TestTempFile(c *C) {
 	f, err := s.FS.TempFile("", "bar")
 	c.Assert(err, IsNil)
 	c.Assert(f.Close(), IsNil)
-	c.Assert(strings.HasPrefix(f.Name(), "bar"), Equals, true)
+
+	c.Assert(strings.Index(f.Name(), "bar"), Not(Equals), -1)
```

Comment:
a temp file has a random part so can't be asset properly. The paths are covered by hundreds of other tests. 

I prefer having this test in this way that having a other test on the temporal package where this test is shadowed.